docker-compose.yml

```diff
@@ -89,6 +89,17 @@ services:
     - "127.0.0.1:6391:6379"
 
 # Dependencies
+  cosmosdb-emulator:
+    image: mcr.microsoft.com/cosmosdb/linux/azure-cosmos-emulator:latest
+    container_name: cosmosdb-emulator
+    environment:
+      - AZURE_COSMOS_EMULATOR_PARTITION_COUNT=2
+      - AZURE_COSMOS_EMULATOR_ENABLE_DATA_PERSISTENCE=false
+      - AZURE_COSMOS_EMULATOR_IP_ADDRESS_OVERRIDE=127.0.0.1
+    ports:
+      - "8081:8081"
+      - "10250-10255:10250-10255"
```

Comment:
Do we _really_ need all these ports 😕 [The docs all say](https://learn.microsoft.com/en-us/azure/cosmos-db/emulator-linux#running) it's port `1234` we need to expose? 🤔 And that's only for the data explorer, so we _probably_ don't even need to do that?
```suggestion
```